sheetScript.gs

```diff
@@ -463,50 +449,34 @@ function deletePlaylistItems(playlistId, deleteBeforeTimestamp) {
   }
 }
 
-function getAllChannelIds_OLD() { // Note: this function is not used.
-  var channelIds = [];
+//
+// Functions for Housekeeping
+// Makes Web App, function call from Google Sheets, add errors, etc
+//
 
-  // First call
-  try {
-    var results = YouTube.Subscriptions.list('snippet', {
-      mine: true,
-      maxResults: 50
-    });
-  } catch (e) {
-    Logger.log("ERROR: " + "Message: [" + e.message + "] Details: " + JSON.stringify(e.details));
-    return;
-  }
-  for (var i = 0; i < results.items.length; i++) {
-    var item = results.items[i];
-    channelIds.push(item.snippet.resourceId.channelId);
+function playlist(pl, sheetID){
```

Comment:
Maybe rename pl to index/row?
```suggestion
// Returns the pl-th playlist ID listed on the sheet
function playlist(pl, sheetID){
````

Reply:
I was actually wondering about this function @victorjzsun. It is not called anywhere. Am I missing the utility of the function?

Reply:
It's part of the deploy as web app function. From the Readme:  
> 6. Usage: append /?pl=N to select the Nth playlist in the spreadsheed. (<- also typo here)

Reply:
Thank you! Mystery solved! I added a comment for that one.